.github/workflows/CompatHelper.yml

```diff
@@ -0,0 +1,17 @@
+name: CompatHelper
+
+on:
+  schedule:
+    - cron: '00 00 * * *'
```

Comment:
This is a personal taste choice, but I tend to randomise the `cron` time, just not to spawn gazillions of jobs at midnight UTC.  I usually do this:
```julia
julia> "$(rand(0:59)) $(rand(0:23)) * * *"
"38 0 * * *"
```

Reply:
Adopted, thanks